src/OrchardCore.Modules/OrchardCore.Resources/Assets.json

```diff
@@ -320,14 +313,6 @@
     "inputs": [
       "node_modules/monaco-editor/min/**/*"
```

Comment:
Instead you should try using the dev.js file and instead of copying it just let it get minified by the gulp pipeline. So we will get ocmonaco.js and ocmonaco.min.js files.